### ISSUE TITLE
Why Twitter?

### ISSUE THREAD

**Issue by ArtexJay** [MehediH/winstall#74]
Why am I forced to log into Twitter for make a pack? 

Seems completely unrelated.

**Comment by hotrungnhan** [comment]
right , twitter f**king block my account , so hope we can login by google , microsoft , or github blabla .

**Comment by typxxi** [comment]
yes, that sucks completely and I would simply expect a Microsoft 365 account to be good enough to do the job or a google account but for sure not Twitter.

**Comment by peppersgc** [comment]
Duplicate of #37

**Comment by st-jliu** [comment]
This project was taken over by Splashtop now, we will continue the development and maintenance, we re-design the whole packs feature, and now we can support both Google/Github/Microsoft login.
You are welcome to provide further feedback and suggestions.
